Apps/WebClient/src/NewClientApp/src/components/site/FooterComponent.vue

```diff
@@ -0,0 +1,75 @@
+<script setup lang="ts">
+import { computed } from "vue";
+
+import HgButtonComponent from "@/components/common/HgButtonComponent.vue";
+import { useAuthStore } from "@/stores/auth";
+import { useConfigStore } from "@/stores/config";
+import { useUserStore } from "@/stores/user";
+
+const authStore = useAuthStore();
+const configStore = useConfigStore();
+const userStore = useUserStore();
+
+const isFooterShown = computed<boolean>(
+    () =>
+        !configStore.isOffline &&
+        (!authStore.oidcIsAuthenticated ||
+            !userStore.isValidIdentityProvider ||
+            userStore.userIsRegistered)
+);
+</script>
+
+<template>
+    <v-footer v-show="isFooterShown" data-testid="footer" color="primary" app>
```

Comment:
I don't see any reason this would benefit from being a `v-show` rather than a `v-if`.